packages/frontend/src/view/components/protocol/EndpointContract.tsx

```diff
@@ -14,12 +17,21 @@ interface Props {
 export function EndpointContract(props: Props): JSX.Element {
   return (
     <ProtocolComponentCard title="Endpoint" subtitle={props.address}>
-      <Row label="Owner" value={props.owner} />
-      <Row label="Default send library" value={props.defaultSendLibrary} />
-      <Row
-        label="Default receive library"
-        value={props.defaultReceiveLibrary}
-      />
+      <ExpandableContainer
+        showText="View contract parameters"
+        hideText="Hide contract parameters"
+      >
+        <Subsection>
+          <Row label="Owner" value={props.owner} />
+          <RowSeparator />
```

Comment:
Row separator could be a border of every non last row (nitpick)

Reply:
[feat: remove row separator](https://github.com/l2beat/lz-monitoring/pull/68/commits/7bf7c42282c2a7bd23211b68135ea4e38ce1a582)